www/install/var/configFileGorgoneTemplate

```diff
@@ -0,0 +1,70 @@
+name: gorgoned-central
+description: Configuration for Central server
+database:
+  db_centreon:
+    dsn: "mysql:host=--ADDRESS--:--DBPORT--;dbname=--CONFDB--"
+    username: "--DBUSER--"
+    password: "--DBPASS--"
+  db_centstorage:
+    dsn: "mysql:host=--ADDRESS--:--DBPORT--;dbname=--STORAGEDB--"
+    username: "--DBUSER--"
+    password: "--DBPASS--"
+gorgonecore:
+  privkey: "--CENTREON_SPOOL--/.gorgone/rsakey.priv.pem"
+  pubkey: "--CENTREON_SPOOL--/.gorgone/rsakey.pub.pem"
+  cipher: "Cipher::AES"
+  keysize: 32
+  vector: 0123456789012345
+modules:
+  - name: httpserver
+    package: "gorgone::modules::core::httpserver::hooks"
+    enable: true
+    address: "--HTTPSERVERADDRESS--"
+    port: "--HTTPSERVERPORT--"
+    ssl: --SSLMODE--
+    auth:
+      enabled: false
+    allowed_hosts:
+      enabled: true
+      subnets:
+        - 127.0.0.1/32
+
+  - name: action
+    package: "gorgone::modules::core::action::hooks"
+    enable: true
+
+  - name: cron
+    package: "gorgone::modules::core::cron::hooks"
+    enable: true
+
+  - name: proxy
+    package: "gorgone::modules::core::proxy::hooks"
+    enable: true
+
+  - name: nodes
+    package: "gorgone::modules::centreon::nodes::hooks"
+    enable: true
+
+  - name: legacycmd
+    package: "gorgone::modules::centreon::legacycmd::hooks"
+    enable: true
+    cmd_file: "--CENTREON_VARLIB--/centcore.cmd"
+    cache_dir: "--CENTREON_CACHEDIR--"
+    cache_dir_trap: "--CENTREON_TRAPDIR--"
+    remote_dir: "--CENTREON_VARLIB--/remote-data/"
+
+  - name: engine
+    package: "gorgone::modules::centreon::engine::hooks"
+    enable: true
+    command_file: "--CENTREON_VARLIB---engine/rw/centengine.cmd"
+
+  - name: broker
+    package: "gorgone::modules::centreon::broker::hooks"
+    enable: false
```

Comment:
@cgagnaire should we enable it by default ?

Reply:
For now we disable it as the "storage" of those stats files needs to change a little bit.

Reply:
It can know.